### PR TITLE
remove kubectl for runtime and dataset controller

### PR DESCRIPTION
**Pull request by xliuqq**
### Ⅰ. Describe what this PR does
Remove kubectl shell command usage for runtime and dataset controller, use client go instead.

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅳ. Describe how to verify it
existed e2e tests.

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/utils/kubeclient/configmap.go [hunk 1]
```diff
@@ -143,3 +143,19 @@ func UpdateConfigMap(client client.Client, cm *v1.ConfigMap) error {
...
+			Name:      name,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			key: string(data),
+		},
+	}
+
+	err = client.Create(context.TODO(), configMap)
+
```
Comment (cheyang) [inline pkg/utils/kubeclient/configmap.go:159]: I think it's better to use this way:

```golang
return client.Create(context.TODO(), configMap)
```

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/utils/kubeclient/configmap.go (hunk 1 above)
Comment (xliuqq) [reply pkg/utils/kubeclient/configmap.go:159]: ok.

**Comment by cheyang** [comment]
I suggest keeping pkg/utils/kubectl/configmap.go in this PR. We can deprecate it in later PR.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**